scripts/run_backend_tests.py

```diff
@@ -548,6 +558,11 @@ def main(args: Optional[List[str]] = None) -> None:
                 and not parsed_args.ignore_coverage):
             raise Exception('Backend test coverage is not 100%')
 
+    if parsed_args.generate_time_report:
+        with utils.open_file(TIME_REPORT_PATH, 'w') as time_report_file:
+            for test_target, time_taken in time_report.items():
+                time_report_file.write('%s:%s\n' % (test_target, time_taken))
+
```

Comment:
Thinking about this, wouldn't it be better to store the results directly as a JSON file and then just load it easily by the check script?

Reply:
Done.